### PR TITLE
cmake,make: bump version and set dev identifier

### PR DESCRIPTION
**Pull request by cspiel1**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -13,18 +13,18 @@
 cmake_minimum_required(VERSION 3.10)
 
 project(re
-  VERSION 2.8.0
+  VERSION 2.9.0
```
Comment (alfredh) [inline CMakeLists.txt:16]: What if the next version is 2.8.1 ?

Comment (sreimers) [reply CMakeLists.txt:16]: This would released from stable branch.

Comment (cspiel1) [reply CMakeLists.txt:16]: So ist 2.9.0 correct?

Comment (sreimers) [reply CMakeLists.txt:16]: 2.9.0-dev should be fine. so we can distinguish the stable release from dev versions

**Pull request merged**